src/tests/functional/plugin/cpu/single_layer_tests/topk.cpp

```diff
@@ -110,63 +216,75 @@ TEST_P(TopKLayerCPUTest, CompareWithRefs) {
     CheckPluginRelatedResults(executableNetwork, "TopK");
 }
 
-const ngraph::element::TypeVector inputPrecisions = {
-    ngraph::element::f32,
+namespace {
+
+const std::vector<ElementType> netPrecisions = {
+    ElementType::f32,
+    ElementType::bf16
+};
+
+std::vector<std::map<std::string, std::string>> additionalConfig = {
+    {{PluginConfigParams::KEY_ENFORCE_BF16, PluginConfigParams::NO}},
+    {{PluginConfigParams::KEY_ENFORCE_BF16, PluginConfigParams::YES}}
 };
 
-const std::vector<int64_t> axes = { 0, 1, 2 };
+const std::vector<int64_t> axes = {0, 1, 2, 3};
+const std::vector<int64_t> k = {1, 5, 7, 18, 21};
 
 const std::vector<ngraph::opset4::TopK::Mode> modes = {
     ngraph::opset4::TopK::Mode::MIN,
     ngraph::opset4::TopK::Mode::MAX
 };
 
 const std::vector<ngraph::opset4::TopK::SortType> sortTypes = {
-    ngraph::opset4::TopK::SortType::SORT_INDICES,
     ngraph::opset4::TopK::SortType::SORT_VALUES,
+    ngraph::opset4::TopK::SortType::SORT_INDICES,
 };
 
-const std::vector<InputShape> inShapes = {
-    InputShape{
-        // dynamic
-        {-1, -1, -1},
-        // target
-        {
-            {10, 13, 11},
-            {15, 10, 5},
-            {5, 5, 8}
-        }
-    },
-    InputShape{
-        // dynamic
-        {-1, { 7, 25 }, -1},
-        // target
-        {
-            {10, 7, 10},
-            {15, 20, 5},
-            {5, 25, 7}
-        }
-    },
-    InputShape{
-        // dynamic
-        {{8, 10}, {7, 17}, {12, 27}},
-        // target
-        {
-            {8, 7, 12},
-            {9, 12, 25},
-            {10, 17, 27}
-        }
-    },
+std::vector<ov::test::InputShape> inputShapes = {
+    {{}, {{21, 21, 21, 21}}},
+};
+
+std::vector<ov::test::InputShape> inputShapesDynamic = {
+    {{21, {20, 25}, 21, {20, 25}}, {{21, 21, 21, 21}, {21, 22, 21, 23}}},
```

Comment:
Fixed. The shapes in topk SLT are relatively large, in order to cover all the cases of different sorting algorithms, which will cost more time. Besides some of the ci machines have more strict total test time limits, like 1 hour. So I had to reduce the total number of test cases. But now I double the dynamic test cases here, since I've halved the test cases according to your another comment about removing redundant bf16 test cases.

Reply:
I have to roll-back. debian_9_arm-cpu_arm ci tests always return "The process was killed by timeout 3000 sec" if I double the dynamic cases. Further discussion is always welcomed. Thanks Dmitry!